src/vs/workbench/contrib/terminalContrib/suggest/browser/terminal.suggest.contribution.ts

```diff
@@ -165,12 +167,18 @@ class TerminalSuggestContribution extends DisposableStore implements ITerminalCo
 	private async _loadLspCompletionAddon(xterm: RawXtermTerminal): Promise<void> {
 		// Load and register the LSP completion providers (one per language server)
 		this._lspModelProvider.value = this._instantiationService.createInstance(LspTerminalModelContentProvider);
-		// TODO: Create with different languages.
+		// TODO: Create with different languages by checking shell type
 		const virtualTerminalDocumentUri = createTerminalLanguageVirtualUri('1', 'py');
-		this._lspModelProvider.value.setContent(virtualTerminalDocumentUri, 'import ast');
+		// this._lspModelProvider.value.setContent(virtualTerminalDocumentUri, 'import ast');
 		const textVirtualModel = await this._textModelService.createModelReference(virtualTerminalDocumentUri);
 		const virtualProviders = this._languageFeaturesService.completionProvider.all(textVirtualModel.object.textEditorModel);
 
+		if (this._lspTerminalDictionaryService) {
+			// set it up so terminal can access the model and set content
+			// TODO: use actual terminal ID
+			this._lspTerminalDictionaryService.set('randomTerminalId', this._lspModelProvider);
```

Comment:
> For this._lspTerminalDictionaryService.set('randomTerminalId', this._lspModelProvider);
Need to actually use real terminal ID for current terminal instance, need to figure out how to get the terminalID